code/svb-docker/python/svb_web/core/views/internal/print_receipt.py

```diff
@@ -0,0 +1,19 @@
+from django.http import HttpResponse, HttpResponseRedirect, HttpResponseBadRequest
+from django.shortcuts import get_object_or_404
+
```

Comment:
You can use [login_required](https://docs.djangoproject.com/en/4.2/topics/auth/default/#django.contrib.auth.decorators.login_required) to only let users who are logged in use this page.

```suggestion
from django.contrib.auth.decorators import login_required
```